raiden/network/transport/matrix/rtc/aiogevent.py

```diff
@@ -0,0 +1,316 @@
+# pylint: skip-file
+# This file is copied from https://github.com/fredo/aiogevent
+# license: https://github.com/fredo/aiogevent/blob/master/COPYING
```

Comment:
We have to ask @jake-bb if linking to the license is enough or if we need to include it somewhere in the repo itself.

Reply:
It's fine as long as the link is alive!

Reply:
@Jake-bb looked over it and according to his opinion it is fine as long as the link exists. That is why I forked it.